src/tiles/admin/StopPlaceTile/components/StopPlacePanel/PanelRow/index.tsx

```diff
@@ -130,19 +139,30 @@ function PanelRow({ stopPlaceId }: { stopPlaceId: string }) {
     }
 
     return (
-        <div className={classes.PanelRow}>
-            <ExpandablePanel className={classes.Expandable} title={header}>
-                <div className={classes.Content}>
-                    {uniqueDepartures.map(({ route }) => (
-                        <RouteCheckbox
-                            key={route}
-                            route={route}
-                            stopPlaceId={stopPlaceWithEstimatedCalls?.id}
-                        />
-                    ))}
-                </div>
-            </ExpandablePanel>
-        </div>
+        <>
+            <div className={classes.PanelRow}>
+                <ExpandablePanel className={classes.Expandable} title={header}>
+                    <div className={classes.Content}>
+                        {uniqueDepartures.map(({ route }) => (
+                            <RouteCheckbox
+                                key={route}
+                                route={route}
+                                stopPlaceId={stopPlaceWithEstimatedCalls?.id}
+                            />
+                        ))}
+                    </div>
+                </ExpandablePanel>
+                {settings.newStops.includes(stopPlaceId) && (
+                    <IconButton
+                        onClick={() => {
+                            handleCloseStopPlaceRow()
+                        }}
```

Comment:
her kan du skrive `onClick={handleCloseStopPlaceRow}` direkte istedet for å wrappe i en ekstra funksjon